### PR TITLE
Remove internal routing from tests

### PR DESCRIPTION
**Pull request by palango**
## Description

Part of #6496

Remove more internal routing from tests.

### CODE REVIEW

File: raiden/tests/utils/transfer.py [hunk 1]
```diff
@@ -115,6 +115,24 @@ def get_channelstate(
     return channel_state
 
 
+def create_route_state_for_route(
+    apps: List[RaidenService], token_address: TokenAddress, fee_estimate: FeeAmount
+) -> RouteState:
+    assert len(apps) > 1, "Need at least two nodes for a route"
+    route = [app.address for app in apps]
```
Comment (ulope) [inline raiden/tests/utils/transfer.py:122]: Would it make sense for this helper to check that there are actually channels between the nodes in the given order?

File: raiden/tests/integration/transfer/test_refundtransfer.py [hunk 2]
```diff
@@ -386,8 +426,25 @@ def test_different_view_of_last_bp_during_unlock(
...
+    )
+    assert token_network
     payment_status = app0.mediated_transfer_async(
-        token_network_address, amount_refund, TargetAddress(app2.address), identifier_refund
+        token_network_address=token_network_address,
+        amount=amount_refund,
+        target=TargetAddress(app2.address),
+        identifier=identifier_refund,
+        route_states=[
+            RouteState(
```
Comment (ulope) [inline raiden/tests/integration/transfer/test_refundtransfer.py:440]: Can't this use the `create_route_state_for_route` helper as well?

File: raiden/tests/integration/transfer/test_refundtransfer.py [hunk 3]
```diff
@@ -594,8 +653,25 @@ def test_refund_transfer_after_2nd_hop(
...
+    )
+    assert token_network
     payment_status = app0.mediated_transfer_async(
-        token_network_address, amount_refund, TargetAddress(app3.address), identifier_refund
+        token_network_address=token_network_address,
+        amount=amount_refund,
+        target=TargetAddress(app3.address),
+        identifier=identifier_refund,
+        route_states=[
+            RouteState(
```
Comment (ulope) [inline raiden/tests/integration/transfer/test_refundtransfer.py:667]: Same as above.

**Review by ulope — APPROVED** [review]
LGTM

A couple of questions below.

File: raiden/tests/integration/transfer/test_refundtransfer.py (hunk 2 above)
Comment (palango) [reply raiden/tests/integration/transfer/test_refundtransfer.py]: Nice find, somehow stepped over this while refactoring.

File: raiden/tests/utils/transfer.py (hunk 1 above)
Comment (palango) [reply raiden/tests/utils/transfer.py:122]: That makes sense. I'll add that in a different PR.

**Pull request merged**